i18n/en.json

```diff
@@ -6456,7 +6456,7 @@
   },
   {
     "id": "store.sql_post.query_max_post_size.max_post_size_bytes",
-    "translation": "Post.Message supports at most %d bytes"
+    "translation": "Post.Message supports at most %d characters (%d bytes)"
```

Comment:
I don't think these numbers will be the same always.

Reply:
In fact I don't believe we have a byte limit on posts (could be anywhere from 1 to 16,000 bytes).